### PR TITLE
NYC Header & Footer

### PR DESCRIPTION
**Pull request by aannyyaa**
Apply CTO house style

### CODE REVIEW

File: src/components/footer.js [hunk 1]
```diff
@@ -51,48 +51,23 @@ function Footer() {
           <Logo
             size="medium"
             image={
-              <img
-                
-                alt={t("footer.secondary.heading.logo")}
-                src={logoImg}
-              />
+              <img alt={t("footer.secondary.heading.logo")} src={logoImg} className="footer__logo" />
```
Comment (saifulislamdev) [inline src/components/footer.js]: Can you think of a better way to arrange the order of attributes?

File: src/components/footer.js [hunk 2]
```diff
@@ -51,48 +51,23 @@ function Footer() {
...
-                <Link key="phone" to="tel:1-800-555-5555">
-                  {t("footer.secondary.address.phone")}
-                </Link>,
-                <Link key="email" to="mailto:info@agency.gov">
-                  {t("footer.secondary.address.email")}
-                </Link>,
-              ]}
-              // role=""
-              // about=""
-            />
```
Comment (saifulislamdev) [inline src/components/footer.js:95]: None of this was needed after all? :/

File: src/components/language-selector.js [hunk 3]
```diff
@@ -1,74 +1,49 @@
 import React, { useEffect, useState } from "react";
 import { navigate } from "gatsby";
-import { Grid, Link, Menu, NavDropDownButton } from "@trussworks/react-uswds";
-import globe from "../images/logos/globe.png";
+import { Grid, Menu, NavDropDownButton } from "@trussworks/react-uswds";
+import { useTranslation } from "react-i18next";
+import FeatherIcon from "feather-icons-react";
+import { Link } from ".";
```
Comment (saifulislamdev) [inline src/components/language-selector.js:5]: The imports can be rearranged better. For example, `import { Link } from ".";` can come immediately after you import the components from `react-uswds`. Maybe, maybe not.

File: src/styles/class-overrides.scss [hunk 4]
```diff
@@ -1,36 +1,37 @@
...
 .usa-graphic-list__heading,
-.usa-hero__heading {
+.usa-hero__heading,
+.usa-intro,
+.usa-prose {
   font-family: family("ui");
 }
 
-.usa-nav__submenu,
-.usa-nav__primary button[aria-expanded="true"],
```
Comment (saifulislamny) [inline src/styles/class-overrides.scss:12]: You didn't delete this line by accident, right?

File: src/styles/class-overrides.scss [hunk 5]
```diff
@@ -1,36 +1,37 @@
...
-    border-style: solid;
-    border-width: $mocto-button-border-width;
-    border-color: color("secondary");
+  button[aria-expanded="false"] {
+    color: color("disabled-dark");
+    &:hover {
+      color: color("base-darker");
+    }
+    &::after {
+      border: $mocto-button-border-width solid color("secondary");
```
Comment (saifulislamdev) [inline src/styles/class-overrides.scss:27]: Excellent!

File: src/styles/class-overrides.scss [hunk 6]
```diff
@@ -123,6 +121,26 @@
...
+.banner__language-selector-label {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+}
+
+.usa-nav__submenu .usa-nav__submenu-item a {
+  text-decoration: none;
+  &:hover {
+    text-decoration: none;
```
Comment (saifulislamdev) [inline src/styles/class-overrides.scss:142]: Do you have to repeat this line?

File: src/components/footer.js (hunk 2 above)
Comment (aannyyaa) [reply src/components/footer.js:95]: Yeah none of it was in the CTO styles - we can ask them if we can keep this section. I know you had to work on this part for a while!

Comment (saifulislamdev) [reply src/components/footer.js:95]: No, it's fine. Lesson learned.

File: src/styles/class-overrides.scss (hunk 6 above)
Comment (aannyyaa) [reply src/styles/class-overrides.scss:142]: Good call - seems like it's not needed here

File: src/components/footer.js (hunk 1 above)
Comment (aannyyaa) [reply src/components/footer.js]: What do you think? Alphabetical?

Comment (saifulislamdev) [reply src/components/footer.js]: Order of importance: className, src, alt

Comment (aannyyaa) [reply src/components/footer.js]: Okay I put class name first in all the components, except for links, which have `to` and sometimes one other attribute (like `onClick`) first

**Review by saifulislamdev — APPROVED** [review]

**Pull request merged**